test/libethereum/State.cpp

```diff
@@ -93,6 +93,8 @@ void doStateTests(json_spirit::mValue& _v, bool _fillin)
 }
 } }// Namespace Close
 
+//Disable OLD StateTests. Use GeneralStateTests at StateTests.cpp instead
+/*
```

Comment:
Please use `//` to make it better visible that the tests are disabled. Why did you disable the tests? Does it make sense to delete them completely?